ts/components/conversation/right-panel/overlay/message-info/components/MessageInfo.tsx

```diff
@@ -57,6 +64,48 @@ const showDebugLog = () => {
   ipcRenderer.send('show-debug-log');
 };
 
+const showDebugMessageInfo = false;
```

Comment:
How do I use this? Is it in the `window` object?

Reply:
set it to true? but don't commit it!

Reply:
ok I'll make this more dynamic